### PR TITLE
Issue Template

### PR DESCRIPTION
**Pull request by SusLes**
Created an Issue Template for .NET Framework Issues.

### CODE REVIEW

File: ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Issue Title
+
+Provide a descriptive title.
+
+# Version info 
+
+Provide the .NET Framework Release key value and your Windows version (output of `winver`).
+
+HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\NET Framework Setup\NDP\v4\Full, Release
```
Comment (rpetrusha) [inline ISSUE_TEMPLATE]: In [How to: Determine Which .NET Framework Versions Are Installed](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#net_b), we document the path as HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full, Release. Are they the same?

File: ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Provide the .NET Framework Release key value and your Windows version (output of `winver`).
+
+HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\NET Framework Setup\NDP\v4\Full, Release 
+
+# General
+
+Provide details on the problem you are experiencing, including step by step instructions to reproduce the issue.
+
+Please provide a code sample for your issue if it is relevant, either inline, link to a gist (or similar) or as a zipped attachment.
```
Comment (rpetrusha) [inline ISSUE_TEMPLATE]: link to a --> as a link to a
comma after "(or similar)"
"please" here or in line 17 isn't necessary.

File: ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+For UI problems please provide screenshots inline or as a zipped attachment.
```
Comment (rpetrusha) [inline ISSUE_TEMPLATE]: comma after "problems"

**Comment by mairaw** [comment]
Closing this as I understand, we don't folks filing issues in this repo.

**Pull request closed without merging**